sympy/physics/_biomechanics/tests/test_tutorial.py

```diff
@@ -0,0 +1,607 @@
+
+    k, c = sm.symbols('k, c')
+    x, v = me.dynamicsymbols('x, v')
+    N = me.ReferenceFrame('N')
+    O, P = me.Point('O'), me.Point('P')
+    P.set_pos(O, x*N.x)
+    P.set_vel(N, v*N.x)
+
+    # loads
+    force_on_P = me.Force(P, -k*P.pos_from(O) - c*P.vel(N))
+    force_on_O = me.Force(O, k*P.pos_from(O) + c*P.vel(N))
+    print('Force objects:')
+    print(force_on_P)
+    print(force_on_O)
+
+    # pathways
+    lpathway = me.LinearPathway(O, P)
+    # TODO : Force could have some methods like .magnitude(), as
+    # force.force.magnitude() isn't obvious.
+    print('Linear pathway loads:')
+    pprint(lpathway.to_loads(-k*x - c*v))
+    lpathway.length
+    lpathway.extension_velocity
+
+    Q, R = me.Point('Q'), me.Point('R')
+    Q.set_pos(O, 1*N.y)
+    R.set_pos(O, 1*N.x + 1*N.y)
+    opathway = me.ObstacleSetPathway(O, Q, R, P)
+    print('Ostacle pathway loads:')
+    pprint(opathway.to_loads(-k*opathway.length))
+
+    # geometry and WrappingGeometry
+    r = sm.symbols('r')
+    theta = me.dynamicsymbols('theta')
+    O, P, Q = sm.symbols('O, P, Q', cls=me.Point)
+    A = me.ReferenceFrame('A')
+    A.orient_axis(N, theta, N.z)
+    P.set_pos(O, r*N.x)
+    Q.set_pos(O, N.z + r*A.x)
+    cyl = me.WrappingCylinder(r, O, N.z)
+    wpathway = me.WrappingPathway(P, Q, cyl)
+    print('Wrapping pathway loads:')
+    pprint(wpathway.to_loads(-k*wpathway.length))
+
+    # actuators
+
+    class SpringDamper(me.ActuatorBase):
+
+        # positive x spring is in tension
+        # negative x spring is in compression
+
+        def __init__(self, P1, P2, spring_constant, damping_constant):
+            self.P1 = P1
+            self.P2 = P2
+            self.k = spring_constant
+            self.c = damping_constant
+
+        def to_loads(self):
+
+            x = self.P2.pos_from(self.P1).magnitude()
+            v = x.diff(me.dynamicsymbols._t)
+
+            xhat = self.P2.pos_from(self.P1).normalize()
+
+            force_P1 = me.Force(self.P1, self.k*x*xhat + self.c*v*xhat)
+            force_P2 = me.Force(self.P2, -self.k*x*xhat - self.c*v*xhat)
+
+            return [force_P1, force_P2]
+
+    spring_damper = SpringDamper(O, P, k, c)
+    print(spring_damper.to_loads())
+
+    class SpringDamper2(me.ForceActuator):
+
+        # positive x spring is in tension
+        # negative x spring is in compression
+
+        def __init__(self, pathway, spring_constant, damping_constant):
+            self.pathway = pathway
+            self.k = spring_constant
+            self.c = damping_constant
+            self.force = (-self.k*pathway.length -
+                          self.c*pathway.extension_velocity)
+
+    spring_damper2 = SpringDamper2(lpathway, k, c)
+    print(spring_damper2.to_loads())
+
+    # activation
+    # TODO : there are no sympy symbols in this activation, I was expecting to
+    # find a symbol for a and e and then something that indicates that a = e.
+    # Also confused to why rhs() returns Matrix(0, 1, []).
```

Comment:
For consistency, all of the `rhs` methods return a column matrix. But for `ZerothOrderActivation` there are no state variables/state equations, so the number of rows here is zero. The mass matrix/`M` attribute returns a `Matrix` with shape (0, 0), while the forcing vector/`F` attribute also returns `Matrix(0, 1, [])`.